code/Language/Drasil/ExprExtract.hs

```diff
@@ -0,0 +1,50 @@
+module Language.Drasil.ExprExtract (egetDoc)where
+
+import Control.Lens ((^.))
+import Language.Drasil.Document
+import Language.Drasil.Chunk.Eq (QDefinition)
+import Language.Drasil.Unit(UnitDefn)
+import Language.Drasil.Chunk.Quantity
+import Language.Drasil.Expr
+import Language.Drasil.Chunk.ShortName
+import Language.Drasil.Classes (HasUID(uid),NamedIdea(term), Idea(getA),
+  HasSymbol(symbol), IsUnit, ExprRelat(relat), HasDerivation(derivations), 
+  HasReference(getReferences), ConceptDomain, Definition(defn))
+
+egetDoc :: Document -> [Expr]
+egetDoc (Document _ _ s) = concatMap egetSec s
+
+-- If collected sentences are used for collecting symbols, 
+-- section collection has to avoid Reference Material section (named "RefMat"),
+-- because this section includes Table of symbol which would
+-- cause loop in collecting.
+
+-- Auxiliary Constants Section (named "AuxConstants") contains standard 
+-- values (like min, max) that are used for defined basic Chunk.
+-- These values should not appear in the basic Table of symbol.
+egetSec :: Section -> [Expr]
+egetSec (Section _ _ _ (ShortNm "RefMat")) = []
+egetSec (Section _ sc _ _) = concatMap egetSecCon sc
+egetSec (Section _ _ _ _) = []
```

Comment:
This line is likely unreachable, as the previous line will always match.

Reply:
Changes were made in 3ecbfc6469af72ea800ba7eddd1a510f5aee2e67.